zombies.js

```diff
@@ -111,6 +150,21 @@
  * @return {boolean} true/false     Whether player was able to store item in pack.
  */
 
+Player.prototype.takeItem = function(item){
+  if (this.getPack().includes(item)){
+    this.getPack().splice(0, 0, item);
+    console.log(`${this.name} picked up ${item.name}`);
+    return true;
+  }
+
+  if (this.getPack().length >= 3){
+    console.log(`${this.name} could not store ${item.name}`);
+    return false;
+  }
+
+  this.getPack().splice(0, 0, item);
```

Comment:
You could just use `this.getPack().push(item)`. This is much easier to read and more intuitive.